db/migrate/20220112214138_create_conversation.rb

```diff
@@ -0,0 +1,10 @@
+class CreateConversation < ActiveRecord::Migration[6.1]
+  def change
+    create_table :conversations do |t|
+      t.integer :user1, null: false
+      t.integer :user2, null: false
```

Comment:
These should be foreign keys, not just integer fields.
```suggestion
     t.references :user1, null: false, foreign_key: { to_table: :users }
      t.references :user2, null: false, foreign_key: { to_table: :users }
```